src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -84,25 +45,13 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         '&copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
     ].join('');
 
-    /**
-     * Variables used for limiting to one isochrone request at a time.
-     * Unlike the planTrip request, the isochrone request cannot be handled
-     * solely via debounce, due to differences in the way the isochrone
-     * request flows through the system. There are actions that take place
-     * on this module (drawing the isochrone on the map), and then actions
-     * that take place on the sidebar module (generating the destinations),
-     * so doing the limiting correctly is more of a challenge.
-     */
-    var activeIsochroneRequest = null;
-    var pendingIsochroneRequest = null;
-
     function MapControl(options) {
         this.events = events;
         this.eventNames = eventNames;
         this.options = $.extend({}, defaults, options);
         overlaysControl = new CAC.Map.OverlaysControl();
         map = new cartodb.L.map(this.options.id, { zoomControl: false })
-            .setView(this.options.center, this.options.zoom);
+                           .setView(this.options.center, this.options.zoom);
```

Comment:
Are you saying this should be further to the right so it aligns under the paren after `map` (as [here] (https://github.com/azavea/cac-tripplanner/commit/ad89bca928bbe6ecabbed8ab2cba7bb622e6bc9e#diff-bfd83b9efe49a865ad1b7a51733c0b66R334) or that it should be back under the `=` where I moved it from?  My understanding is that when breaking lines on a chained method call it's good to align the dots, though in this case `setView` is the first method on `map`, so it's true that lining it up under the `L.map` dot isn't quite correct.